### DISCUSSION TITLE
Questions re outline & fill stacked

### DISCUSSION THREAD

**Discussion by marshallapparalautomation** [inkstitch/inkstitch#1972] [General]
I have a logo that's 2 vector layers stacked and trying to sort out best method to fill and underlay. I added the 2nd layer just outlined to assist with clean edges but cannot sort out the best method to have a consistent zig zag underlay based on either layer. I appreciate any thoughts. 

Other questions: How can I add trims?

Attached is the vector .svg
![Southern BotanicalFillaStroke](https://user-images.githubusercontent.com/68432172/202726242-57054ab6-8bc4-4428-9617-6137053cc0d0.svg)

**Comment by kaalleen** [comment]
How big do you want the design to be in the end? From the file I would think that you would prefer to use a satin stitch (https://inkstitch.org/docs/stitches/satin-column/) and not a fill.

You can add trims like any other command: https://inkstitch.org/docs/commands/#attach-visual-commands-to-objects

* Select the element after which you want the thread to be cut
* Go to `Extensions > Ink/Stitch > Commands > Attach commands to selected objects ...
* Select the trim command
* Click on apply

**Comment by marshallapparalautomation** [comment]
I would love to use a satin stitch however cutting the ends of the paths and adding the nodes to have similar amounts on both parts is too time consuming unless you know of a simpler method. My "cheat" is to have two layers top being the last stitch fill and middle being an extra self created underlay as running stitch. The pattern runs same angle direction but at least looks decent with exception of material peaking through parts which is why I asked about the underlay suggested setting.

When I check the underlay on the top fill layer the underlay misses a large part of the fill. I don't mind it not being zig zag any pattern would be fine. Can you recommend best param for underlay on the fill ?

**Comment by kaalleen** [comment]
> adding the nodes to have similar amounts on both parts is too time consuming unless you know of a simpler method.

Use rungs. This is faster and easier to handle. The node method isn't really useful here.
Still of course I know that turning a font into satin is time consuming. But the result will be much, much better.

**Reply by marshallapparalautomation** [reply]
I have some ideas how ink stitch can generate "rungs automatically" and thus create satin stitches. Im also struggling with driving the command line of your extension. Could we setup a zoom call and talk about potentially how I could help inkstitch and you can help me accomplish a clients immediate needs?
